script.js

```diff
@@ -0,0 +1,74 @@
+const books = [
+  {
+    id: 1,
+    title: 'book 1',
+    author: 'author 1',
+  },
+  {
+    id: 2,
+    title: 'book 2',
+    author: 'author 2',
+  },
+  {
+    id: 3,
+    title: 'book 3',
+    author: 'author 3',
+  },
+  {
+    id: 4,
+    title: 'book 4',
+    author: 'author 4',
+  },
+];
```

Comment:
- [optional] I think you can remove this template since you are no longer using it for this milestone I suppose. This will make your codebase cleaner and avoid confusion in the future. ℹ️